common/src/main/java/com/wynntils/models/items/properties/DurabilityItemProperty.java

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
This is what `DurableItemProperty` does. Perhaps not the best of names, though. But can you please rename that interface instead of creating a new?

Reply:
I am blind